platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/Feature.kt

```diff
@@ -33,11 +36,19 @@ class Feature(
         get() {
             if (manifest?.thumbnail == null) return null
             val entry = content.getEntry(manifest.thumbnail) ?: return null
-            val options = BitmapFactory.Options()
-            // For now, we assume all thumbnails are xhdpi, i.e. 2x scale factor
-            options.inDensity = DisplayMetrics.DENSITY_XHIGH
-            content.getInputStream(entry).use {
-                return BitmapFactory.decodeStream(it, null, options)
+            if (entry.name.endsWith(".pdf")) {
```

Comment:
There's probably a product decision somewhere that includes this, but I haven't been able to find it in Swift (maybe done natively) and second, bitmaps are fixed size, and they are probably not going to be adapted to screen orientation, so not sure how accessible they're going to be.

Reply:
same is done for iOS -> https://github.com/polypoly-eu/polyPod/blob/6e18e8ffe93f9e291bfdb93b75c17d038df7994e/platform/ios/PolyPodApp/PolyPod/HomeScreenView.swift#L59;

`bitmapFromPdf` takes care of properly scaling the Bitmap according to screen density.

Reply:
As said, there's a product decision somewhere, but it would be much easier, scalable and everything if it was created in a properly scalable format. I can imagine PDF features, once adapted, being too small to actually be seen, which is why I mention accessibility.
I guess that since this is for preview and might include graphics or logos and things like that, it's going to be easier to re-use a PDF instead of exporting them to a scalable format. Also, PDFs in the repo... Difficult to say whether they have the right license or not, so care needs to be taken to import them during build

> This is probably already taken care of, but just in case.

And then the "about" needs to reflect the correct (c) for the PDFs.

This is not something strictly within the scope of this PR, but it might be within the scope to preview adding copyright notices to the app just in case.

Reply:
I don't quite understand what you mean, we use pdf's so we can properly scale the images.

Reply:
It's not so much the scaling what worries me but the fitting; my point being that the original PDF might include some features that, scaled down, will not be visible. Also the license thing, which is probably more in scope.

Reply:
pardon my ignorance, AFAIK  but PDFs work the same way as SVG's in terms of scaling, and also it is an improvement on top of old way of using single scale PNGs.
What do you exactly mean by license, and why raise this problem now when using PDFs?

Reply:
Because I don't really know the reason why PDFs are chosen, and I assume it's because it's copy produced elsewhere. When we include SVGs or images in the repo, we take good care of heeding their licenses. If that's the case also here, no big deal.

Reply:
PDFs is the usual format used for iOS; Displaying SVG's natively on iOS requires bringing external dependencies, same is true for Android; Pdfs can be rendered easily to an image.

Previously, native apps(iOS and Android) used single scale images, we did decide to use something like PDFs or SVGs to improve image quality;

As for licensing, I will have to check.